crates/abi/src/lib.rs

```diff
@@ -52,6 +52,11 @@ pub fn miralis_end_benchmark() -> ! {
     }
 }
 
+/// Ask Miralis to lock the payload
+pub fn lock_payload() {
+    unsafe { miralis_ecall(abi_protect_payload::MIRALIS_PROTECT_PAYLOAD_LOCK_FID).ok() };
+}
+
```

Comment:
Since we created a new EID for the policy we can't use `miralis_ecall` (because it uses the Miralis EID). Let's create a new function `policy_ecall` which is similar to `miralis_ecall` but also takes the policy EID as a parameter, and use it for the `lock_payload` ^^